### PR TITLE
APIS-677: Change delimiter & add 'Copy to clipboard' button

### PR DESCRIPTION
**Pull request by rm-hull**

### CODE REVIEW

File: app/views/developers/developers.scala.html [hunk 1]
```diff
@@ -101,12 +101,44 @@ <h1 class="flush--top">Developers</h1>
 
     @if(users.length > 0) {
     <div class="form-group>">
-        <p><a class="button" href="mailto:?bcc=@emails">Email @{users.length} developers</a></p>
+        <p>
+            <a class="button" href="mailto:?bcc=@emails" title="Opens external mail client BCC'ing all @{users.length} email addresses.">Email @{users.length} developers</a>
+            <a class="button" href="#" title="Copies all @{users.length} email addresses into the system clipboard." onclick="copyTextToClipboard('@emails'); return false;">Copy to clipboard</a>
```
Comment (antroy) [inline app/views/developers/developers.scala.html]: Why does the onclick handler return false?

File: app/views/developers/developers.scala.html [hunk 2]
```diff
@@ -101,12 +101,44 @@ <h1 class="flush--top">Developers</h1>
...
+        </p>
     </div>
     }
     <div class="form-group>">
         <a class="back-to-top-link" href="#">Back to Top</a>
     </div>
+
+    <script type="text/javascript">
```
Comment (antroy) [inline app/views/developers/developers.scala.html]: Is there nothing in any of the js libraries we use that does this?

File: app/views/developers/developers.scala.html (hunk 1 above)
Comment (rm-hull) [inline app/views/developers/developers.scala.html]: Returning false cancels out the click action (i.e. it prevents following through to the # anchor). See http://stackoverflow.com/questions/128923/whats-the-effect-of-adding-return-false-to-an-click-event-listener for some background

File: app/views/developers/developers.scala.html (hunk 2 above)
Comment (rm-hull) [inline app/views/developers/developers.scala.html]: Nothing on public; was some stuff on private but it looked like it used some Adobe Flash abomination to do the clipboard copy-paste mechanics

**Pull request merged**